front/src/modules/settings/components/IconWithLabel.tsx

```diff
@@ -0,0 +1,45 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { IconComponent } from '@/ui/icon/types/IconComponent';
+
+type IconWithLabelProps = {
+  Icon: IconComponent;
+  label: string;
+};
+
+const StyledContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(3)};
+  padding: ${({ theme }) => theme.spacing(1)};
+`;
+
+const StyledSubContainer = styled.div`
+  align-items: center;
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(1)};
+`;
+const StyledItemLabel = styled.div`
+  color: ${({ theme }) => theme.font.color.secondary};
+  font-size: ${({ theme }) => theme.font.size.sm};
+  font-style: normal;
+  font-weight: ${({ theme }) => theme.font.size.md};
+  leading-trim: both;
+  line-height: ${({ theme }) => theme.text.lineHeight.md};
+  text-edge: cap;
```

Comment:
introducing fancy css properties ;)

Reply:
It was on the figma file aha, should I let them in?